frontend/src/pages/News/SinglePost.tsx

```diff
@@ -307,13 +327,12 @@ export default function SinglePost() {
         </article>
 
         {/* PREV / NEXT ARTICLES NAV */}
-        <div className='mx-10 grid max-w-md grid-cols-1 items-center justify-center border-0 border-t border-solid border-altGrey pt-24 md:grid-cols-3'>
+        <div className='mx-10 mb-10 pt-10 grid max-w-md grid-cols-1 items-center justify-center border-0 border-t border-solid border-altGrey md:grid-cols-3'>
           {prevArticle && (
             <HetPaginationButton
               direction='previous'
-              onClick={() => {
-                goPrevious()
-              }}
+              onClick={goPrevious}
+              data-pagination-content='true'
             >
               {getHtml(prevArticle?.title?.rendered ?? '')}
```

Comment:
we should refactor this in the future... basically the "title" coming from wordpress _should_ be a string, but sometimes authors will use like bold or italics in the title in which case it ends up sending a string of html. I think we can refactor this to just simply extract the actual text from wordpress, and then you can just apply the truncation directly here (rather than needing the useEffect and then manually manipulating the dom elements to shorten them). But for now this works even if it's more complex than I wish it had to be haha. Nice work!